python/ngsi_v2/.openapi-generator-ignore

```diff
@@ -0,0 +1,23 @@
+# OpenAPI Generator Ignore
+# Generated by openapi-generator https://github.com/openapitools/openapi-generator
+
+# Use this file to prevent files from being overwritten by the generator.
+# The patterns follow closely to .gitignore or .dockerignore.
+
+# As an example, the C# client generator defines ApiClient.cs.
+# You can make changes and tell OpenAPI Generator to ignore just this file by uncommenting the following line:
+**/api_client.py
```

Comment:
Let's add a note here to remind us why we're not regenerating this file:

    # We modified `api_client.py` so we have to exclude it from code generation.
    # In fact, we had to suppress the content type header in GET requests to avoid API errors.

Reply:
fixed in https://github.com/orchestracities/sdk/pull/1/commits/9785d793a6da33549815722ba4ec0437b592048c